lib/matplotlib/widgets.py

```diff
@@ -1930,8 +1935,9 @@ def press(self, event):
             key = event.key or ''
             key = key.replace('ctrl', 'control')
             # move state is locked in on a button press
-            if key == self.state_modifier_keys['move']:
-                self._state.add('move')
+            for state in ['move']:
+                if key == self._state_modifier_keys[state]:
+                    self._state.add(state)
```

Comment:
Why this change? It's functionally the same but more complex code.

Reply:
No reason, this is a left over from a previous change, which I forgot to update, this is fixed now.